assets/blocks/course-outline/course-block/edit.js

```diff
@@ -102,35 +100,4 @@ const selectors = ( select ) => ( {
 	structure: select( COURSE_STORE ).getStructure(),
 } );
 
-const dispatches = ( dispatch, ownProps, { select } ) => {
-	const editPost = select( 'core/edit-post' );
-	const { toggleEditorPanelEnabled } = dispatch( 'core/edit-post' );
-
-	const hasLegacyMetaBoxes =
-		editPost.isEditorPanelEnabled( 'meta-box-course-lessons' ) ||
-		editPost.isEditorPanelEnabled( 'meta-box-module_course_mb' );
-
-	const toggleLegacyMetaboxes = ( enable ) => {
-		// Side-effect - Prevent submit modules.
-		document
-			.querySelectorAll( '#module_course_mb input' )
-			.forEach( ( input ) => {
-				input.disabled = ! enable;
-			} );
```

Comment:
I think this got left behind, should still be needed, right?

Reply:
Oops! Good catch! I updated this commit: https://github.com/Automattic/sensei/pull/3661/commits/13ef04bde0d8ce029ac0f236f1cd78a39b284c12